site/_docs/encodings.md

```diff
@@ -122,6 +132,12 @@ DIRECT        | PRESENT         | Yes      | Boolean RLE
 DIRECT_V2     | PRESENT         | Yes      | Boolean RLE
               | DATA            | No       | Unbounded base 128 varints
               | SECONDARY       | No       | Unsigned Integer RLE v2
+DECIMAL_V1    | PRESENT         | Yes      | Boolean RLE
+              | DATA            | No       | Signed Integer RLE v1
+              | SECONDARY       | Yes      | Signed Integer RLE v1
```

Comment:
We probably need a RLE128 that can just encode a int128 directly. Then we can just use the DATA stream directly.

Reply:
We may use LEB128. Thoughts?